jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1010,6 +1011,39 @@ properties:
             type: number
             fits_keyword: YOFFSET
             blend_table: True
+          starting_set:
+            title: Starting 4-point set number
+            type: integer
+            enum: [1, 3, 5]
+            fits_keyword: DSETSTRT
+          number_of_sets:
+            title: Total number of 4-point sets
+            type: integer
+            enum: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+            fits_keyword: NUMDSETS
+          direction:
+            title: Direction of dither pattern offsets
+            type: string
+            enum: [NEGATIVE, POSITIVE]
```

Comment:
The keyword dictionary has an explicit `NULL` value in the enum list for this and other keywords.  Do we need that?  I assumed that it would be represented in by a None in Python.

Reply:
They probably have the NULL values because some exposures won't use these attributes and hence they won't have a value. We need to determine exactly what value SDP will populate each keyword with in those cases. It can't be the string "NULL" for keywords that have int or float data types.

Reply:
I suspect the keyword will not get populated if it doesn't apply to the data?

Reply:
Sent email asking for clarification.